traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java

```diff
@@ -63,7 +63,11 @@ private Traveler(String apiKey, Context applicationContext, boolean isSandboxMod
             @Override
             protected void main() {
                 if (authTokenFetchTask.getError() != null) {
-                    throw new RuntimeException(String.format("Could not initialize Traveler SDK. %s ", authTokenFetchTask.getError()));
+                    // Auth token failed, effectively means initialization never happened.
+                    Traveler.this.session = null;
+                    Traveler.this.applicationContext = null;
+                    localInstance = null;
+                    Log.e(TAG, String.format("Could not initialize Traveler SDK. %s ", authTokenFetchTask.getError()));
```

Comment:
@ata-n This fixes the runtimeException when calling Traveler.initialize while having no internet, but I would prefer to add a "InitializationCallback {success(), error(reason)}" to notify UI, how's that sound?

Reply:
hmmm...I dont like informing anyone about this...best course of action is just to be silent

Reply:
You dont need to nullify everything...do you?

Reply:
I removed the log

Upon `authTokenFetchTask.getError` we don't have a valid session.. so  definitely `localInstance` should be nulled, otherwise other Traveler methods would think we have a valid session and start making failing calls.

I don't think it makes sense to have a state where `localinstance` is null, but `applicationContext` and `session` have values. The SDK-user will eventually call Traveler.initialize again, and `session` and `applicationContext` will be overwritten anyway...